File: app/src/main/java/com/tangem/presentation/dialog/RootFoundDialog.java
package com.tangem.presentation.dialog;

import android.app.AlertDialog;
import android.app.Dialog;
import android.app.DialogFragment;
import android.os.Bundle;

import com.tangem.wallet.R;

public class RootFoundDialog extends DialogFragment {
    public static final String TAG = RootFoundDialog.class.getSimpleName();

    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        return new AlertDialog.Builder(getActivity())
                .setIcon(R.drawable.tangem_logo_small_new)
                .setTitle(R.string.device_is_rooted)
                .setCancelable(false)
                .setPositiveButton(R.string.got_it, null)
                .create();
    }

}

File: app/src/main/java/com/tangem/presentation/dialog/RootFoundDialog.kt
package com.tangem.presentation.dialog

import android.app.AlertDialog
import android.app.Dialog
import android.os.Bundle
import android.support.v4.app.DialogFragment

import com.tangem.wallet.R

class RootFoundDialog : DialogFragment() {

    companion object {
        val TAG: String = RootFoundDialog::class.java.simpleName
    }

    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        return AlertDialog.Builder(activity)
                .setIcon(R.drawable.tangem_logo_small_new)
                .setTitle(R.string.device_is_rooted)
                .setCancelable(false)
                .setPositiveButton(R.string.got_it, null)
                .create()
    }

}